t/tnt/init.lua

```diff
@@ -77,5 +205,9 @@ return {
         return data
     end,
 
-    cfg = tnt_prepare
+    cfg = tnt_prepare,
+    cfg_cluster = tnt_cluster_prepare,
```

Comment:
Maybe move all the cluster stuff to a separate table? Then the call would look like `tnt.cluster.cfg`, `tnt.cluster.connect_replica`, ...

Reply:
moved to tnt.cluster